### PR TITLE
i fixed #305. Removed some hyphens. boom.

### LINKED ISSUE #305: Breadcrumb Switcher
The breadcrumb switcher is a new component that will allow the user no navigate between multiple views at the same level in the information hierarchy. This is useful, when browsing details of objects, for example, as it allows the user to navigate to a new object without the need to return first to a higher level summary view.

**Use Cases:** As a user, I want to quickly navigate to view another object at a peer level in the information hierarchy.

**References:** [JIRA Story](https://patternfly.atlassian.net/browse/OSUX-62), [Preliminary mockup](https://drive.google.com/open?id=0B5KC3vr5EkgmQlpaaE9KTC15Q3M)

**Products Needing This:** Red Hat Satellite, Red Hat Storage Console

**Scope of work:** TBD

**Stakeholders/Reviewers:** Serena Doyle, Roxanne Hoover, Matt Carrano

**Other Info:** 
- Roxanne has spoken to someone in Satellite who is going to contribute this to React PatternFly repo
- Roxanne has spoken to Walden Raines in Satellite who is going to contribute this to the Angular PatternFly repo

### PR DESCRIPTION
**Pull request by jeperry**

### CODE REVIEW

**Comment by LHinson** [comment]
This fixes #205

**Comment by LHinson** [comment]
+1 looks good bitmoji

File: pattern-library/cards/trend-card/design/design.md [hunk 1]
```diff
@@ -5,62 +5,63 @@
...
+    Use a tooltip on hover to display additional information about exact points on the sparkline (e.g. values and/or percentages). A vertical line and marker helps to accentuate the hover point.
 
   1. **Time Frame** (optional)**:**
-    - The time frame is indicated by placing a label on one of the following areas:
+    The time frame is indicated by placing a label on one of the following areas:
       - Under the title
       - Under the sparkline
       - In a [time frame filter](https://www.patternfly.org/pattern-library/dashboard/dashboard-card/#example-overview-1)
-    - The time frame on a card is optional since there could be a single instance of the time frame label or time frame filter that applies to all cards within a single view.
+
```
Comment (Rohoover) [inline pattern-library/cards/trend-card/design/design.md]: Is this needed? Extra "+"

**Comment by priley86** [comment]
+1 bitmoji

**Pull request merged**